source/openstack/index.rst

```diff
@@ -0,0 +1,14 @@
+.. _x86:
+
+OpenStack (x86_64)
+===============
+
+(description)
```

Comment:
What should I put here?

Reply:
Basically something about how we host three different OpenStack clusters for each architecture. The x86_64 one is is used typically for production hosting while the other two are for supporting development/CI for those archs. And then something about these docs are provided to help users interact with our clusters.